gr-utils/read_file_metadata/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+########################################################################
+# Install python files and apps
+########################################################################
+include(GrPython)
+
+GR_PYTHON_INSTALL(
```

Comment:
I'm wondering if read_file_metadata since it's only a single python script should be a top level (same as modtool), or there should be a "misc" directory for this and other random utilities.  That way each new python utility, which it would be nice to have more, wouldn't require its own directory and cmake file

Reply:
Yeah, I thought so. Especially, it can be burdensome that every additional single-file tool requires another `CMakeLists.txt`.
Then again, a `misc` folder is not much better than `utils` (current situation), and the only tool for now is `read_file_metadata`.
So I think it's better to leave this PR as it is now, and if this snowballs in the future, adopt your solution.